### DISCUSSION TITLE
Reviewing pull requests if the "Add a review" link doesn't appear

### DISCUSSION THREAD

**Discussion by pth6570** [earthlab-education/Earth-Analytics-2023-01-Intro#199] [Q&A01 Online Portfolio] [answered]
We discussed this question during our discussion section this afternoon --  Just posting the answer here in case anyone else had the same question. 
 
**Question**: When first looking at a pull request under the “Conversations” tab, there’s a link to click to “Add your review.” Once at least one review was completed, it was replaced by text saying “Show All Reviewers” and there’s no longer a link to add a review.
 
**Answer**: If you go to the “Files Changed” tab, you can click a button on the right of the screen to “Review Changes" and then add your review.

**Comment by eculler** [comment, marked as answer]
Answer: If you go to the “Files Changed” tab, you can click a button on the right of the screen to “Review Changes" and then add your review.

Thanks, @pth6570 !
